### PR TITLE
0.14.0 Changelog

### PR DESCRIPTION
**Pull request by OpenSauce04**

### CODE REVIEW

File: .github/docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+
+* Removed unused `env_effects.glsl` file
+
+* Updated authors
+
+* Credits loading message replaced by a message that links to the website's credits page
+
+* Added a new preset toggle to control whether or not the underwater distortion effect is enabled
+  
+  * This toggle is disabled on the `Lowest` quality setting
```
Comment (jebbyk) [inline .github/docs/CHANGELOG.md]: I've enabled it even on lowest settings cuz I can't notice any difference switching animations on/off even on lowes settings and 225x400 resolution (less pixels processing with the same vertexs count)

Comment (OpenSauce04) [reply .github/docs/CHANGELOG.md]: Ok, I'll update the changelog to reflect this

**Comment by jebbyk** [comment]
how you are makin PRs "work in progress"

**Comment by jebbyk** [comment]
@OpenSauce04  How you are making PRs to be "work in progress"? I need this option :)

**Comment by OpenSauce04** [comment]
> @OpenSauce04  How you are making PRs to be "work in progress"? I need this option :)

It's just an option when you are initially creating your PR.
Just press the arrow next to the Create Pull Request button and select "Create Draft Pull Request"

**Pull request merged**